python/ray/actor.py

```diff
@@ -246,8 +246,19 @@ def __call__(self, *args, **kwargs):
     def _from_modified_class(cls, modified_class, class_id,
                              max_reconstructions, num_cpus, num_gpus, memory,
                              object_store_memory, resources):
+
+        # Make sure the actor class we are constructing inherits from the
+        # original class so it retains all class properties.
+        class DerivedActorClass(cls, modified_class):
```

Comment:
I think the inheritance order should be reversed as we want the `modified_class` to take precedence, right? We might also want to check and log a warning if we're overwriting any existing properties.

Reply:
I think cls should take precedence (otherwise the actor won't be useable), but I'm also hiding the attributes now and warn if class attributes don't get overwritten, so we don't get into the situation where we need to choose between the two :)